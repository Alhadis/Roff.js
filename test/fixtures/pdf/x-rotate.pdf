%PDF-1.4
%‚„œ”
3 0 obj << /Contents [4 0 R  ]
/Group << /CS /DeviceRGB
/S /Transparency
>>
/Parent 2 0 R
/Type /Page
>>
endobj
4 0 obj << /Length 3762
>>
stream
q 1 0 0 1 0 0 cm
1 J
1 j
% V12000
%% H72000
% DFd
0 g
% h72000
% n12000 0
q BT
1 0 0 1 144 780 Tm

0 Tc
% V12000
%% H72000
% n12000 0
1 0 0 1 72 780 Tm
0 Tc
% V12000
%% H72000
% DFg 0
1 0 0 1 72 780 Tm
0 Tc
ET Q
0 g
% h36000
% v36000
q BT
1 0 0 1 108 780 Tm

0 Tc
% md
1 0 0 1 108 744 Tm
0 Tc
0 g
% s10000
% DP -6336 -3816 2520 -2520
ET Q
0 G
108 744 m
101.664 747.816 l
104.184 750.336 l
f
% n12000 0
q BT
1 0 0 1 104.184 750.336 Tm
0 Tc
% V12000
%% H72000
% Dt 100 0
1 0 0 1 72 780 Tm
0 Tc
ET Q
0.1 w
% H72000
% n12000 0
q BT
1 0 0 1 72 780 Tm
0 Tc
% V48000
%% H108000
% Dp -6336 -3816 2520 -2520
1 0 0 1 108 744 Tm
0 Tc
ET Q
108 744 m
101.664 747.816 l
104.184 750.336 l
s
% n12000 0
q BT
1 0 0 1 104.184 750.336 Tm
0 Tc
% V12000
%% H72000
% Dt -1000 0
1 0 0 1 72 780 Tm
0 Tc
ET Q
0.4 w
% h1000
% n12000 0
q BT
1 0 0 1 73.4 780 Tm
0 Tc
% V12000
%% H72000
% Dl 30888 30888
1 0 0 1 72 780 Tm
0 Tc
ET Q
72 780 m
102.888 749.112 l
s
% n12000 0
q BT
1 0 0 1 102.888 749.112 Tm
0 Tc
% V30000
%% H90000
% x font 5 TR
% f5
% V30000
1 0 0 1 90 762 Tm
0 Tc
/F5 10 Tf
%% H90000
% x X ps: exec gsave currentpoint 2 copy translate 35.0000 rotate neg exch neg exch translate
%% n12000 0
1 0 0 1 90 762 Tm
0 Tc
ET Q
q
0.819 -0.574 0.574 0.819 -420.789 189.428 cm
% n12000 0
q BT
1 0 0 1 90 762 Tm
/F5 10 Tf
0 Tc
% V32200
%% H74165
% tCaption
1 0 0 1 74.165 759.8 Tm
0 Tc
% == 'Caption'=31.67 'xpos=105.835
% --- wht=2500, pend=0, nomv=0
% n12000 0
0 Tw (Caption) Tj
% V12000
%% H72000
% V12000
1 0 0 1 72 780 Tm
0 Tc
%% H72000
% x X ps: exec grestore
%% n12000 0
1 0 0 1 72 780 Tm
0 Tc
ET Q
Q
% n12000 0
q BT
1 0 0 1 72 780 Tm
/F5 10 Tf
0 Tc
% V12000
%% H108000
% DP 6336 3816 -2520 2520
1 0 0 1 108 780 Tm
0 Tc
ET Q
108 780 m
114.336 776.184 l
111.816 773.664 l
f
% n12000 0
q BT
1 0 0 1 111.816 773.664 Tm
/F5 10 Tf
0 Tc
% V12000
%% H72000
% Dt 100 0
1 0 0 1 72 780 Tm
0 Tc
ET Q
0.1 w
% H72000
% n12000 0
q BT
1 0 0 1 72 780 Tm
/F5 10 Tf
0 Tc
% V12000
%% H108000
% Dp 6336 3816 -2520 2520
1 0 0 1 108 780 Tm
0 Tc
ET Q
108 780 m
114.336 776.184 l
111.816 773.664 l
s
% n12000 0
q BT
1 0 0 1 111.816 773.664 Tm
/F5 10 Tf
0 Tc
% V12000
%% H72000
% Dt -1000 0
1 0 0 1 72 780 Tm
0 Tc
ET Q
0.4 w
% h1000
% n12000 0
q BT
1 0 0 1 73.4 780 Tm
/F5 10 Tf
0 Tc
% 
% n12000 0
% V30000
%% H126000
% V30000
1 0 0 1 126 762 Tm
0 Tc
%% H126000
% x Slant 50
% x X ps: exec gsave currentpoint 2 copy translate -135.0000 rotate neg exch neg exch translate
%% n12000 0
1 0 1.192 1 126 762 Tm
0 Tc
ET Q
q
-0.707 0.707 -0.707 -0.707 753.912 1211.719 cm
% n12000 0
q BT
1 0 1.192 1 126 762 Tm
/F5 10 Tf
0 Tc
% V48000
%% H144000
% cA
1 0 1.192 1 144 744 Tm
0 Tc
% == 'A'=7.22 'xpos=151.22
% --- wht=2500, pend=0, nomv=0
% 
% Dl -30888 -30888
0 Tw [(A) 722 ] TJ
ET Q
144 744 m
113.112 774.888 l
s
% 
% V32200
%% H111075
% tF
q BT
1 0 1.192 1 111.075 759.8 Tm
/F5 10 Tf
0 Tc
% == 'F'=5.56 'xpos=116.635
% --- wht=2500, pend=0, nomv=0
% H116485
0 Tw (F) Tj
5.410 0 Td
% tooBar
% == 'ooBar'=24.44 'xpos=140.925
% --- wht=2500, pend=0, nomv=0
% cZ
% == 'Z'=6.11 'xpos=147.035
% --- wht=2500, pend=0, nomv=0
% 
% n12000 0
0 Tw [(ooBarZ) 611 ] TJ
% V12000
%% H72000
% cA
1 0 1.192 1 72 780 Tm
0 Tc
% == 'A'=7.22 'xpos=79.22
% --- wht=2500, pend=0, nomv=0
% 
% Dp 0 -36000 -108000 0 0 36000
0 Tw [(A) 722 ] TJ
ET Q
72 780 m
72 816 l
-36 816 l
-36 780 l
s
% cZ
q BT
1 0 1.192 1 -36 780 Tm
/F5 10 Tf
0 Tc
% == 'Z'=6.11 'xpos=-29.89
% --- wht=2500, pend=0, nomv=0
% 
% V12000
0 Tw [(Z) 611 ] TJ
%% H72000
% x X ps: exec grestore
%% n12000 0
1 0 1.192 1 72 780 Tm
0 Tc
ET Q
Q
% n12000 0
q BT
1 0 1.192 1 72 780 Tm
/F5 10 Tf
0 Tc
% h-20000
% tyea
% == 'yea'=13.88 'xpos=65.88
% --- wht=2500, pend=-20000, nomv=0
% x trailer
% V792000
0 Tw [2000 (yea) ] TJ
%% x stop
% x stop
1 0 1.192 1 65.88 0 Tm
0 Tc
ET Q
Q
endstream
endobj
1 0 obj << /Pages 2 0 R 
/Type /Catalog
>>
endobj
8 0 obj << /CreationDate (D:20170704173425+10'00')
/Creator (groff version 1.22.3)
/ModDate (D:20170704173425+10'00')
/Producer (gropdf version 1.22.3)
>>
endobj
5 0 obj << /BaseFont /Times-Roman
/Encoding 6 0 R 
/Subtype /Type1
/ToUnicode 7 0 R 
/Type /Font
>>
endobj
6 0 obj << /Differences [0 /asciicircum /asciitilde /Scaron /Zcaron /scaron /zcaron /Ydieresis /trademark /quotesingle /Euro /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /space /exclam /quotedbl /numbersign /dollar /percent /ampersand /quoteright /parenleft /parenright /asterisk /plus /comma /hyphen /period /slash /zero /one /two /three /four /five /six /seven /eight /nine /colon /semicolon /less /equal /greater /question /at /A /B /C /D /E /F /G /H /I /J /K /L /M /N /O /P /Q /R /S /T /U /V /W /X /Y /Z /bracketleft /backslash /bracketright /circumflex /underscore /quoteleft /a /b /c /d /e /f /g /h /i /j /k /l /m /n /o /p /q /r /s /t /u /v /w /x /y /z /braceleft /bar /braceright /tilde /.notdef /quotesinglbase /guillemotleft /guillemotright /bullet /florin /fraction /perthousand /dagger /daggerdbl /endash /emdash /.notdef /fi /fl /.notdef /.notdef /dotlessi /.notdef /grave /hungarumlaut /dotaccent /breve /caron /ring /ogonek /quotedblleft /quotedblright /oe /lslash /quotedblbase /OE /Lslash /.notdef /exclamdown /cent /sterling /currency /yen /brokenbar /section /dieresis /copyright /ordfeminine /guilsinglleft /logicalnot /minus /registered /macron /degree /plusminus /twosuperior /threesuperior /acute /mu /paragraph /periodcentered /cedilla /onesuperior /ordmasculine /guilsinglright /onequarter /onehalf /threequarters /questiondown /Agrave /Aacute /Acircumflex /Atilde /Adieresis /Aring /AE /Ccedilla /Egrave /Eacute /Ecircumflex /Edieresis /Igrave /Iacute /Icircumflex /Idieresis /Eth /Ntilde /Ograve /Oacute /Ocircumflex /Otilde /Odieresis /multiply /Oslash /Ugrave /Uacute /Ucircumflex /Udieresis /Yacute /Thorn /germandbls /agrave /aacute /acircumflex /atilde /adieresis /aring /ae /ccedilla /egrave /eacute /ecircumflex /edieresis /igrave /iacute /icircumflex /idieresis /eth /ntilde /ograve /oacute /ocircumflex /otilde /odieresis /divide /oslash /ugrave /uacute /ucircumflex /udieresis /yacute /thorn /ydieresis /ellipsis ]
/Type /Encoding
>>
endobj
7 0 obj << /Length 426
>>
stream
/CIDInit /ProcSet findresource begin
12 dict begin
begincmap
/CIDSystemInfo
<< /Registry (Adobe)
/Ordering (UCS)
/Supplement 0
>> def
/CMapName /Adobe-Identity-UCS def
/CMapType 2 def
1 begincodespacerange
<0000> <FFFF>
endcodespacerange
2 beginbfrange
<008b> <008f> [<00660066> <00660069> <0066006c> <006600660069> <00660066006C>]
<00ad> <00ad> <002d>
endbfrange
endcmap
CMapName currentdict /CMap defineresource pop
end
end
endstream
endobj
2 0 obj << /Count 1
/Kids [3 0 R  ]
/MediaBox [0 0 612 792 ]
/Resources << /Font << /F5 5 0 R 
>>
/ProcSet [/PDF /Text /ImageB /ImageC /ImageI ]
>>
/Rotate 0
/Type /Pages
>>
endobj
xref
0 9
0000000000 65535 f 
0000003940 00000 n 
0000006865 00000 n 
0000000015 00000 n 
0000000127 00000 n 
0000004152 00000 n 
0000004259 00000 n 
0000006389 00000 n 
0000003990 00000 n 
trailer
<<
/Info 8 0 R 
/Root 1 0 R
/Size 9
>>
startxref
7046
%%EOF
