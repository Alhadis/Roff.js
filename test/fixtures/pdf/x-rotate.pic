.PS
arrow down right aligned "Caption"
move right
arrow up left aligned "FooBar"
.PE
