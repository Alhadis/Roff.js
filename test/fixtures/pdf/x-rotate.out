x T pdf
x res 72000 1 1
x init
x F fixtures/x.pic
p1
V12000
H72000
DFd
h72000
n12000 0
V12000
H72000
n12000 0
V12000
H72000
DFg 0
h36000
v36000
md
s10000
DP -6336 -3816 2520 -2520
n12000 0
V12000
H72000
Dt 100 0
H72000
n12000 0
V48000
H108000
Dp -6336 -3816 2520 -2520
n12000 0
V12000
H72000
Dt -1000 0
h1000
n12000 0
V12000
H72000
Dl 30888 30888
n12000 0
V30000
H90000
x font 5 TR
f5
V30000
H90000
x X ps: exec gsave currentpoint 2 copy translate 35.0000 rotate neg exch neg exch translate
n12000 0
V32200
H74165
tCaption
n12000 0
V12000
H72000
V12000
H72000
x X ps: exec grestore
n12000 0
V12000
H108000
DP 6336 3816 -2520 2520
n12000 0
V12000
H72000
Dt 100 0
H72000
n12000 0
V12000
H108000
Dp 6336 3816 -2520 2520
n12000 0
V12000
H72000
Dt -1000 0
h1000
n12000 0

n12000 0
V30000
H126000
V30000
H126000
x Slant 50
x X ps: exec gsave currentpoint 2 copy translate -135.0000 rotate neg exch neg exch translate
n12000 0
V48000
H144000
cA
Dl -30888 -30888

V32200
H111075
tF
H116485
tooBar
cZ
n12000 0
V12000
H72000
cA
Dp 0 -36000 -108000 0 0 36000
cZ
V12000
H72000
x X ps: exec grestore
n12000 0
h-20000
tyea
x trailer
V792000
x stop
